config/initializers/prisons.rb

```diff
@@ -1,5 +1,9 @@
 Rails.configuration.prisons = []
 
 Dir["config/prisons/*.yml"].each { |file|
-  Prison.create(YAML.load_file(file))
+  prison = YAML.load_file(file)
+  unless Rails.env.production?
+    prison['email'] = "pvb.#{prison['nomis_id']}@maildrop.dsd.io"
```

Comment:
It was line of least resistance.  Using names means having to parameterize...Can change it once I've got a Site association in place.  This was just to unblock @threedaymonk on testing on preprod.